src/main.cpp

```diff
@@ -4,6 +4,8 @@
 #include <docopt/docopt.h>
 #include <spdlog/spdlog.h>
 
```

Comment:
```suggestion

// This file will be generated automatically when you run the CMake configuration step.
// It creates a namespace called `myproject`.
// You can modify the source template at `configured_files/config.hpp.in`.
```
This comment should not be necessary for many of our users, but I think something like this could be helpful for those who are new to CMake.